democracy_club/apps/hermes/templates/hermes/posts_for_tag.html

```diff
@@ -1,7 +1,7 @@
 {% load thumbnail %}
 {% load typogrify_tags %}
 {% if posts %}
-    <h2>On the blog</h2>
+    {% comment %} <h2>On the blog</h2> {% endcomment %}
```

Comment:
I'd like to use this functionality to DRY up the template but it doesn't work if we keep this header or the link to all posts below.

Reply:
@Bekabyx any thoughts on this? This template is used as an `includes` in a few spots.